Tools/Pipeline/Gtk/MainWindow.cs

```diff
@@ -742,11 +742,26 @@ public void OnRenameActionActivated (object sender, EventArgs e)
             UpdateMenus();
             expand = false;
         }
-        
+
         public void OnDeleteActionActivated (object sender, EventArgs e)
         {
-            projectview1.Remove ();
-            UpdateMenus();
+            var dialog = new MessageDialog(this, DialogFlags.Modal, MessageType.Warning, ButtonsType.None, "How do you want to remove selected file/s?");
+            dialog.Title = "Delete File/s";
+            dialog.SecondaryText = "Do note that selecting \"Delete\" option will delete the file and this action will not be undoable.";
+
+            dialog.AddButton("Delete", ResponseType.Apply);
+            dialog.AddButton("Cancel", ResponseType.Cancel);
+            dialog.AddButton("Remove", ResponseType.Ok);
```

Comment:
The alternative implementation is having a separate Delete and Remove commands sort of like VS.

I do worry that mixing them both together like this... then having a 3 button prompt... could lead to costly mistakes by users.

A remove which works without prompts and a delete that asks a "Are you sure this is not undoable"... seems like a safer approach.

@dellis1972 @KonajuGames ?